examples/create-react-app-with-typescript/public/index.html

```diff
@@ -1,16 +1,15 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no">
-    <!-- PWA primary color -->
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

Comment:
`minimum-scale=1` is important for GPU rasterization.

Reply:
Was this manually added or previously default in create-react-app?

Reply:
manually added